core/server/index.js

```diff
@@ -78,8 +79,8 @@ function init(options) {
         debug('Themes & apps done');
 
         models.init();
-        // @TODO: this is temporary, replace migrations with a warning if a DB exists
-        return db.bootUp();
+    }).then(function () {
+        return knexMigrator.isDatabaseOK();
```

Comment:
### current state: 
When Ghost starts, Ghost asks `knex-migrator` if the database is OK.
`return knexMigrator.isDatabaseOK()` 
If the database is not OK, Ghost will die and tell you what todo (we forward the error from `knex-migrator`).

###  that is the following disadvantages:
When your database was never initialised, you need to run `knex-migrator init` before your can start `node index.js`. So one more manual step if you don't use Ghost-CLI.

Ghost-CLI will initialise your database automatically before it starts Ghost, see https://github.com/TryGhost/Ghost-CLI/pull/57/files#diff-0b441864c5280e9e0a69a0d9e97f7d86R50.

### my thinking now is
Why not put `knexMigrator.init` into Ghost directly?

```
knexMigrator.isDatabaseOK()
  .catch(function () {
    return knexMigrator.init();
});
```